NASA-AMMOS/slim
# Documentation Starter Kits: GitHub Issue Templates

## Purpose
- Starter kits for GitHub issue templates (bug, new features)
## Proposed Changes
- [ADD] Bug report templates (markdown & form)
- [ADD] New feature request templates (markdown & form)
## Testing
- See demo of GitHub Markdown template samples [here](https://github.com/riverma/terraformly/issues/new/choose)
- See demo of GitHub Form template samples [here](https://github.com/nasa/opera-sds-pcm/issues/new/choose)
